heuristics/uc_bids.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+def create_key(template, outtype=('nii.gz',), annotation_classes=None):
+    if template is None or not template:
+        raise ValueError('Template must be a valid format string')
+    return template, outtype, annotation_classes
+
+def infotodict(seqinfo):
+    """Heuristic evaluator for determining which runs belong where
+    
+    allowed template fields - follow python string module: 
+    
+    item: index within category 
+    subject: participant id 
+    seqitem: run number during scanning
+    subindex: sub index within group
+    """
+    t1w = create_key('anat/sub-{subject}_T1w')
+    t2w = create_key('anat/sub-{subject}_acq-{acq}_T2w')
+    flair = create_key('anat/sub-{subject}_acq-{acq}_FLAIR')
+    rest = create_key('func/sub-{subject}_task-rest_acq-{acq}_run-{item:02d}_bold')
+
+    info = {t1w: [], t2w: [], flair: [], rest: []}
+
+    for idx, seq in enumerate(seqinfo):
+        x,y,z,n_vol,protocol,dcm_dir = (seq[6], seq[7], seq[8], seq[9], seq[12], seq[3])
```

Comment:
@danlurie - i just asked @mgxd to update existing examples to use named tuple. you can literally do: `seq.dim1` instead of `seq[6]` for example. if in the future column ordering changes the heuristics would still work.